### ISSUE TITLE
Evaluation 

### ISSUE THREAD

**Issue by deepmeng** [maudzung/YOLO3D-YOLOv4-PyTorch#3]
hi, Nguyen. Thanks for your great work. 
I loaded the well-trained model which was trained after 300 epochs and I evaluated it. I converted the prediction results to the format which is same as the kitti label and I evaluated the prediction results  using the 'kitti eval' tool. Could you please help me check whether this result is credible? I used 1481 samples which was used in 'val' stage for the evaluation. The obtained results are as follows:

```
 > number of files for evaluation: 1481
 >   done.
 >                car_detection AP: 51.263809 48.594360 54.498703
 >             car_orientation AP: 51.238495 48.424850 54.232555
 >   pedestrian_detection AP: 23.030304 27.769840 31.115997
 > pedestrian_orientation AP: 20.538658 24.933500 27.936405
 >          cyclist_detection AP: 41.100662 43.356640 44.776119
 >        cyclist_orientation AP: 40.850254 42.822056 43.808239
 > 
 >             car_detection_ground AP: 63.355881 65.948364 71.283600
 > pedestrian_detection_ground AP: 28.163477 33.746151 37.436134
 >        cyclist_detection_ground AP: 41.011620 39.616348 40.990261
 > 
 > Eval 3D bounding boxes
 > car_detection_3d AP: 60.061161 62.226303 67.980537
 > pedestrian_detection_3d AP: 27.972027 33.435966 37.111938
 > cyclist_detection_3d AP: 40.705566 39.199333 40.584419
 
```

**Comment by maudzung** [comment]
Hi @deepmeng 

You need to make sure that the ground-truth objects and predicted objects are in the same range. Please check the configs [here](https://github.com/maudzung/YOLO3D-YOLOv4-PyTorch/blob/4e7f65fe25198f2202f8317fa88321729821ceb8/src/config/kitti_config.py#L14)

**Comment by deepmeng** [comment]
I think you're right. The metrics of the 'kitti eval' tool are not appropriate for this work.
